kong/init.lua

```diff
@@ -856,6 +860,11 @@ function Kong.init_worker()
   if kong.clustering then
     kong.clustering:init_worker()
   end
+
+  ok, err = wasm.init_worker()
+  if not ok then
+    stash_init_worker_error(err)
```

Comment:
```suggestion
    stash_init_worker_error("failed to load wasm filters: " .. err)
    return
```

- Missing prefix (is it on purpose?)
- It is not guaranteed this chunk will be the last chunk in this function forever.

Reply:
Thanks, I missed the `return` on the first update.

https://github.com/Kong/kong/pull/11248